### ISSUE TITLE
Host Screen Capture

### ISSUE THREAD

**Issue by githubja-lab** [kasmtech/workspaces-issues#619]
We are seeking a way to natively capture and share the host machine's screen within a container session in Kasm. This feature would greatly enhance workflow efficiency during remote demonstrations or troubleshooting, eliminating the need for external screen-sharing tools.

Currently, we are using OBS with the virtual camera option through Kasm’s webcam passthrough. However, this method is inefficient and introduces extra steps, such as starting the virtual camera and then opening another application like MS Teams or Zoom inside of the Kasm workspace for sharing. Additionally, the resolution is limited to 720p, making text difficult to read and reducing clarity.

A native screen capture and sharing feature would significantly improve Kasm Workspaces for collaboration and troubleshooting. Ideally, it would support high-quality, low-latency screen sharing, with easy user control based on session needs.

Another example is Google Remote Desktop, which provides seamless screen sharing through a web application. A similar solution integrated into Kasm would be highly beneficial.

Thank you.

**Comment by j-travis** [comment]
Have you tried the session sharing feature? 
https://kasmweb.com/docs/latest/guide/session_sharing.html#session-sharing

**Comment by githubja-lab** [comment]
Yes, that works for sharing Docker sessions within Kasm agents, but I want to share my local host desktop or application (Mac/Linux/Windows). To do this, I need to capture the desktop/application content and stream it into the session.

We’ve considered using OBS with the virtual camera, which allows us to capture the desktop and create the virtual camera that we can send to a Kasm session thanks to the webcam passthrough. In this setup, my desktop becomes the virtual camera, and we can use a meeting app like MS Teams inside a Kasm session to share the webcam feed.

However, we're looking for a more streamlined solution that removes the need for OBS, allowing us to share the local desktop with less overhead as well maybe removing the needs of Teams

**Comment by teja-kasm** [comment]
We’re currently enhancing our GitHub issue tracker to improve organization and efficiency.

As part of this process, we’re closing older issues to help keep the tracker focused on current and active topics.

If this issue is still relevant or you’re still experiencing the problem, please feel free to create a new issue.

Thank you for your understanding!
